sandbox/src/App.js

```diff
@@ -28,92 +28,97 @@ import RedirectedNewPage from "./RedirectedNewPage";
 import PersonalizationAnalyticsClientSide from "./PersonalizationAnalyticsClientSide";
 import PersonalizationFormBased from "./PersonalizationFormBased";
 import Identity from "./Identity";
+import AlloyVersion from "./components/AlloyVersion";
 
 function BasicExample() {
   return (
-    <Router>
```

Comment:
This diff looks scary and hard to read, so here's a simplified version.

If went from 

```html
<Router>
  …
</Router>
```

to this

```js
<>
  <Router>
    …
  </Router>
  <AlloyVersion/>
<>